lib/fog/compute/google/requests/insert_route.rb

```diff
@@ -2,32 +2,31 @@ module Fog
   module Compute
     class Google
       class Mock
-        def insert_route(_name, _network, _dest_range, _priority, _options = {})
+        def insert_route(_route_name, _network, _dest_range, _priority, _options = {})
           Fog::Mock.not_implemented
         end
       end
 
       class Real
-        def insert_route(name, network, dest_range, priority, options = {})
-          api_method = @compute.routes.insert
-          parameters = {
-            "project" => @project
-          }
-          body_object = {
-            "name" => name,
-            "network" => network,
-            "destRange" => dest_range,
-            "priority" => priority
-          }
-          body_object["description"] = options[:description] if options[:description]
-          unless options[:tags].nil? || options[:tags].empty?
-            body_object["tags"] = options[:tags]
-          end
-          body_object["nextHopInstance"] = options[:next_hop_instance] if options[:next_hop_instance]
-          body_object["nextHopGateway"] = options[:next_hop_gateway] if options[:next_hop_gateway]
-          body_object["nextHopIp"] = options[:next_hop_ip] if options[:next_hop_ip]
+        # Creates a Route resource.
+        #
+        # @see https://cloud.google.com/compute/docs/reference/latest/routes/insert
+        def insert_route(identity, network, dest_range, priority, options = {})
```

Comment:
the mock and read here look different? Maybe just change identity to route_name?